apertium-cat.cat.dix

```diff
@@ -5121,6 +5121,13 @@
 
 </pardef>
 
+<pardef n="semid/éu__n">
+  <e>       <p><l>éu</l>         <r>u<s n="n"/><s n="m"/><s n="sg"/></r></p></e>
+  <e>       <p><l>essa</l>      <r>u<s n="n"/><s n="f"/><s n="sg"/></r></p></e>
```

Comment:
deessa: `<p><l>eessa</l>` and deesses...

Reply:
Thanks.